.ci_support/linux_python2.7.yaml

```diff
@@ -5,7 +5,7 @@ c_compiler_version:
 channel_sources:
 - conda-forge,defaults
 channel_targets:
-- conda-forge main
+- conda-forge rc
```

Comment:
I guess that the cfep-05 text suggests `rc_<package-name>`, so this should be `rc_bottleneck`, right? This avoids problems of installing more stuff from other `rc`s labels b/c conda label are "channels."

Reply:
Thanks, the discussion around the format here made it a bit unclear what the correct value was. Updated to your suggestion.